### PR TITLE
Added an addActivity Command

### PR DESCRIPTION
**Pull request by JKupoluyi**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/EditCommandParser.java [hunk 1]
```diff
@@ -39,7 +39,7 @@ public EditCommand parse(String args) throws ParseException {
         try {
             index = ParserUtil.parseIndex(argMultimap.getPreamble());
         } catch (ParseException pe) {
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditCommand.MESSAGE_USAGE), pe);
```
Comment (Sameow) [inline src/main/java/seedu/address/logic/parser/EditCommandParser.java]: why did u remove pe? doesnt it contain useful information on why it errored?

File: src/test/java/seedu/address/ui/MainWindowCloseTest.java [hunk 2]
```diff
@@ -50,21 +51,28 @@ public void setUp() throws Exception {
...
 
+    /*
     @Test
-    public void close_externalRequest_exitAppRequestEventPosted() {
+    public void close_externalRequest_e
+
+    x
+
+
+    itAppRequestEventPosted() {
```
Comment (Sameow) [inline src/test/java/seedu/address/ui/MainWindowCloseTest.java:65]: lol wats happening here

File: src/test/java/systemtests/HelpCommandSystemTest.java [hunk 3]
```diff
@@ -84,10 +84,15 @@ public void help_multipleCommands_onlyOneHelpWindowOpen() {
...
+    }*/
 
     /**
-     * Asserts that the help window is open, and closes it after checking.
+     * Asserts that the help window is
+     *
+     *
+     *
+     *
+     * open, and closes it after checking.
```
Comment (Sameow) [inline src/test/java/systemtests/HelpCommandSystemTest.java]: why so many spaces

**Review by Sameow — CHANGES_REQUESTED** [review]
nice

File: src/test/java/systemtests/HelpCommandSystemTest.java (hunk 3 above)
Comment (JKupoluyi) [reply src/test/java/systemtests/HelpCommandSystemTest.java]: Spaces were created when running tests on intellij for some reason.

**Review by Sameow — APPROVED** [review]

**Pull request merged**